oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -170,9 +172,11 @@ storage:
     metadataQueryMaxSize: ${SW_STORAGE_MYSQL_QUERY_MAX_SIZE:5000}
     maxSizeOfArrayColumn: ${SW_STORAGE_MAX_SIZE_OF_ARRAY_COLUMN:20}
     numOfSearchableValuesPerTag: ${SW_STORAGE_NUM_OF_SEARCHABLE_VALUES_PER_TAG:2}
+    maxSizeOfBatchSql: ${SW_STORAGE_MAX_SIZE_OF_BATCH_SQL:2000}
+    asyncBatchPersistentPoolSize: ${SW_STORAGE_ASYNC_BATCH_PERSISTENT_POOL_SIZE:4}
   tidb:
     properties:
-      jdbcUrl: ${SW_JDBC_URL:"jdbc:mysql://localhost:4000/tidbswtest"}
+      jdbcUrl: ${SW_JDBC_URL:"jdbc:mysql://localhost:4000/tidbswtest?rewriteBatchedStatements=true"}
```

Comment:
`rewriteBatchedStatements=true` What does this mean? And this doesn't exist in your doc change. Please make sure they are consistent.

Reply:
I have verified in the tidb writing scenario, only by setting the parameters **rewriteBatchedStatements=true**  can the performance of batch execution be improved.

this  explain：
The rewriteBatchedStatements parameter must be added to the url of the MySQL JDBC connection. MySQL JDBC driver ignores the executeBatch() statement by default, splits a set of SQL statements that we expect to be executed in batches, and sends them to the MySQL database one by one. The batch insert is actually a single insert, which directly results in lower performance.Only by setting the rewriteBatchedStatements parameter to true, the driver will help you execute SQL in batches

Reply:
Also, this URL suffix is missing in the `backend-storage.md`.

Reply:
> Then, whether we should add them to MySQL too at least

It is applicable to both MySQL and TiDB(and other databases, of course), you can also check it in the ['best practice' part](https://docs.pingcap.com/tidb/stable/java-app-best-practices#use-batch-api) of TiDB website.

I would recommend setting it for MySQL, too.

Reply:
@chenyi19851209 Please follow the above recommendation.

Reply:
yes. i add this in MYSQL storage.